python/pyspark/sql/connect/session.py

```diff
@@ -326,6 +327,28 @@ def readStream(self) -> "DataStreamReader":
 
     readStream.__doc__ = PySparkSession.readStream.__doc__
 
+    def registerProgressHandler(self, handler: "ProgressHandler") -> None:
```

Comment:
nit but API should be put in `python/docs/source/reference/pyspark.sql/spark_session.rst`

Reply:
Done